lib/models/device.coffee

```diff
@@ -1345,6 +1345,45 @@ getDeviceModel = (deps, opts) ->
 						force: Boolean(options.force)
 		.asCallback(callback)
 
+	###*
+	# @summary Get the supervisor state on a device
+	# @name getSupervisorState
+	# @public
+	# @function
+	# @memberof balena.models.device
+	#
+	# @param {String|Number} uuidOrId - device uuid (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6');
+	#
+	# @example
+	# balena.models.device.getSupervisorState(123);
+	#
+	# @example
+	# balena.models.device.getSupervisorState('7cf02a6',
+	# function(error) {
```

Comment:
Let's not add a newline here and keep the oneline format that the rest methods use.